tests/test_LBFGS_minimization_problems.py

```diff
@@ -27,22 +27,14 @@
 
 import numpy as np
 import scipy.optimize
-from NuMPI.Optimization.LBFGS_Matrix_H import LBFGS
+from NuMPI.Optimization import LBFGS
+from NuMPI import MPI
 import tests.minimization_problems as mp
 
 import pytest
-@pytest.mark.parametrize("Objective",[mp.Extended_Rosenbrock]) # Only these where Solution is analytically known
-@pytest.mark.parametrize("n",[10,20])
-def test_analytical_min(Objective,n):
```

Comment:
Why did you delete this test?

Reply:
because it is a duplicate of test_MPI_LBFGS:test_analytical_min

Up to the number of gridpoints however. I can try to add n=10 and 20 as fixture.

Also the LBFGS tests are split into a lot of different files, maybe I can get that bettter organized.

Reply:
Okay! Please merge this PR